CotEditor/en-GB.lproj/ReportTemplate.md

```diff
@@ -0,0 +1,25 @@
+
+Fill the following template, and post it on [GitHub Issues](https://github.com/coteditor/CotEditor/issues) or send to <coteditor.github@gmail.com>. Please note that the contents of the sent email can be shared on the Issue page. Please write the contents either in English or in Japanese.
+
+-----------------------------------------------
+
+## Environment
+
+- CotEditor: %SHORT_VERSION% (%BUNDLE_VERSION%)
+- System: macOS %SYSTEM_VERSION%
+- Language: English-GB
```

Comment:
Prefer using the macOS-flavored expression "English (UK)" instead of "English-GB".

Reply:
Fixed in d375f85